pkgs/applications/misc/wofi-wifi-menu/default.nix

```
@@ -0,0 +1,43 @@
+{ stdenv, lib, fetchFromGitHub, makeWrapper, wofi, networkmanager, bc, wirelesstools }:
+
+stdenv.mkDerivation rec {
+  pname = "wofi-wifi-menu";
+  version = "0.18";
+
+  src = fetchFromGitHub {
+    owner = "fourstepper";
+    repo = pname;
```

```suggestion
    repo = "wofi-wifi-menu";
```
This has better support for overrides. If users change the pname, it should not affect `src.repo`.